src/components/shared/MarkerTooltipContents.js

```diff
@@ -34,6 +47,50 @@ function getMarkerDetails(data: MarkerPayload): React$Element<*> | null {
           </div>
         );
       }
+      case 'GCMinor': {
+        if (data.nursery && data.nursery.reason) {
```

Comment:
Is there anything that Flow can do to ensure that the data in the profile is valid? For example, if I have:

 
    export type GCMinorMarkerPayload = {
      type: 'GCMinor',
      startTime: Milliseconds,
      endTime: Milliseconds,
      nursery: {|
        reason: string,
      |},
    };

Then I can remove the if statement that checks if data.nursery and data.nursery.reason is present.  However I know that data.nursery is optional, Is there anything to check that the type declaration is correct or that the .json data is correct, and give the user a reasonable error message rather than failing to render the tool-tip?